manifests/extract.pp

```diff
@@ -47,9 +48,19 @@
   case $ensure {
     'present': {
 
+      $default_tar_command = $::osfamily ? {
```

Comment:
You might aswell use this selector in the parameter definition directly (line 39)